include/up-cpp/communication/Subscriber.h

```diff
@@ -43,17 +43,18 @@ struct Subscriber {
 	/// @returns * A unique_ptr to a Subscriber if the callback was
 	///            successfully registered.
 	///          * A UStatus with the appropriate failure code otherwise.
-	[[nodiscard]] utils::Expected<std::unique_ptr<Subscriber>, v1::UStatus>
+	[[nodiscard]] static utils::Expected<std::unique_ptr<Subscriber>,
+	                                     v1::UStatus>
 	subscribe(std::shared_ptr<transport::UTransport> transport,
```

Comment:
Minor: In other headers, we've used something like this to improve readability:

```suggestion
	using SubscriberOrStatus =
		utils::Expected<std::unique_ptr<Subscriber>, v1::UStatus>;

	[[nodiscard]] SubscriberOrStatus subscribe(
		std::shared_ptr<transport::UTransport> transport,
```
... although I'm not sure how much better that is. I wouldn't hold the PR up over this :sweat_smile:

Reply:
Done. I agree it makes code more readable